test/CloudBoilerplateNet.Tests/Services/ReactiveCacheManagerTests.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+using CloudBoilerplateNet.Helpers;
+using CloudBoilerplateNet.Models;
+using CloudBoilerplateNet.Services;
+using KenticoCloud.Delivery;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace CloudBoilerplateNet.Tests.Services
+{
+    public class ReactiveCacheManagerTests
```

Comment:
add a test which would test that an item is retrieved from a cache and not from the web if it is already cached